### PR TITLE
RD-1701 RD-2083 Switched table cell to semantic-ui-react, added stories

### PR DESCRIPTION
**Pull request by Blewin**
## Description

Tickets were requesting features in table.cell that were being added each time manually,
i discovered that table.cell from semantic-ui has most of needed features, so i made our table.data use table.cell from semantic
i added missing feature as verticalAligment="flexMiddle"

## Screenshots / Videos

image

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

storybook updated, with screenshots, tested in stage
## Documentation

storybook updated

### CODE REVIEW

**Review by qooban — COMMENTED** [review]
Is it ready for review? I see CI failing, so probably more changes coming, right?

**Review by qooban — COMMENTED** [review]
Good that you found a way to reuse already existing component exposing alignment props we needed 👍🏻 

Added some comments.

And have one general comment: I would expect more verbosity in PR description - right now it's less of an issue, because we are talking about your changes on Slack, but 
- in the next months it'll change for sure and not only me will be involved in reviews, 
- I'm involved in few code reviews per day so it's really helpful when you see summary of changes and assumptions taken during implementation in one place

thus I would like to encourage you to provide more descriptions 🙂 

In specific:
- in Title I see RD-1801 - probably you meant RD-1701? 🙂  
- in Description section you can add info about chosen path mentioning that you used `Table.Cell` component instead of customizing `td`, so it was possible to reuse already existing props for alignments, how that solved original issues described in RD-1801 and RD-2083 etc.
- in Screenshots / Videos you could show the same screenshot as in added story or just write that reviewer can check it there; 
- in Tests you could write if you have already checked the changes with Stage/Composer
- in Documentation - I guess that added story covers what you changed, so that could also be mentioned, or maybe we should mention somewhere that DataTable.Data has right now the same props as SUIR's `Table.Cell`?

In general - our template is to serve us and not be a bureaucratic nonsense 🙂 I believe that it usually takes few minutes to fill it (so it's not big effort), and it can save reviewers time as well as save our time in the future when we dig into details about why and what someone did 🙂 You already saw example of that (in https://github.com/cloudify-cosmo/cloudify-stage/pull/498 - it helped us to understand what to check when doing changes around widget maximization)

File: test/__snapshots__/HtmlSnapshots.test.ts.snap [hunk 1]
```diff
@@ -972,69 +972,99 @@ exports[`Storyshots Data/DataTable Action Buttons 1`] = `
           className=""
           style={{}}
         >
-          <td>
+          <td
+            className=""
+          >
```
[the remark this answers is not in the document]
Comment (Blewin) [reply test/__snapshots__/HtmlSnapshots.test.ts.snap:977]: its how they do it :(

File: src/components/data/DataTable/TableDataCell.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.verticallyAlignedCell {
```
[the remark this answers is not in the document]
Comment (Blewin) [reply src/components/data/DataTable/TableDataCell.css:1]: now it is :)

File: src/components/data/DataTable/DataTable.stories.tsx [hunk 3]
```diff
@@ -296,68 +227,52 @@ export const expandableRow = () => {
 
     return (
         <DataTable>
-            {/* @ts-expect-error TS(2339) FIXME: Property 'Column' does not exist on type 'typeof D... Remove this comment to see the full error message */}
             <DataTable.Column label="Name" name="id" width="40%" />
-            {/* @ts-expect-error TS(2339) FIXME: Property 'Column' does not exist on type 'typeof D... Remove this comment to see the full error message */}
+
```
[the remark this answers is not in the document]
Comment (Blewin) [reply src/components/data/DataTable/DataTable.stories.tsx]: i hoped prettier would handle it :D

**Review by qooban — APPROVED** [review]
Looks very good 👍🏻

**Review by qooban — APPROVED** [review]

**Pull request merged**